packages/core/strapi/lib/commands/transfer/utils.js

```diff
@@ -121,6 +126,52 @@ const validateExcludeOnly = (command) => {
   }
 };
 
+const logger = createLogger(createOutputFileConfiguration(`import_error_log_${Date.now()}.log`));
```

Comment:
2 things
- It creates a file named import_error_log even for export operations
- It creates the error_log file even if there is no error